UPGRADE.md

```diff
@@ -1,5 +1,19 @@
 # Upgrade
 
+## unreleased
+
+### Add RestRoutingBundle
+
+To make the update to symfony 5 as seamless as possible a new bundle need to be registered in your `config/bundles.php`:
+
+```php
+HandcraftedInTheAlps\RestRoutingBundle\RestRoutingBundle::class => ['all' => true],
+```
+
+### CSV Handler changed
+
+The Sulu `CsvHandler::createResponse` now accept a `ViewHandlerInterface` instead of `ViewHandler`.
```

Comment:
Does that have any effect on the developer applying this update? I guess the `ViewHandler` implements the `ViewHandlerInterface`, so old code would still be working.

Reply:
Type changes are bc break in inheritance so if somebody extended from CsvHandler and overwrite the createResponse he need to change this typehint.

Reply:
Have you ever needed to do that? I think that is very unlikely, and if we document every single change like that, we would end up with a huge `UPGRADE.md`, nobody ready carefully anymore :confused:

Reply:
Ok for me will remove it.